docs/01-engage/02-accounts/02-creating-accounts/index.md

```diff
@@ -0,0 +1,113 @@
+- By default, create new account screen always opens for an individual account.
+- Sometimes, while entering account information, background search might show previously entered accounts with a similar input.   
+:::
+
+1. In the *Name* section, specify the **Title**, **First Name** and **Last Name**.
+
+![Create Account Name](./create-account-1.png)
+
+2. Next, in the *Address* section, enter the **Country** you reside in, and it's respective **Postcode**. Input your address as **Line 1**, **Line 2** or **Line 3** and the **Town/City**.
+
+:::info
+- Most of the time, the address may appear relative to the postcode search.
+- For some countries, address input parameters might be different.
+:::
+
+![Create Account Address](./create-account-2.png)
+
+3. Moving forward, in the *Phone Number* section, enter your phone number along with the country code. Then select whether the number is a *fixed line* or *mobile*. You can also add extra phone numbers with **+ Add Phone Number** and remove them with the **Remove** button. 
+
+:::caution
+For incorrect and incomplete phone numbers, a **validation pop-up error message** will appear!
+:::
+
+![Create Account Phone Number](./create-account-3.png)
+
+4. In the *Email Address* section, input your *Personal* or *Work* email and select it respectively. You can also add extra email addresses with **+ Add Email Address** and remove them with the **Remove** button.
+
+![Create Account Email](./create-account-4.png)
+
+5. After this, in the *Online Presence* section, select the type from the drop-down including *Facebook*, *LinkedIn* and *Instagram* etc and state its example. You can add additional types using **Add Profile** and remove them as wanted with the **Remove** option.
+
+:::note 
+Online Presence refers to the combined existence of an individual or organization that is found online via search. 
+:::
+
+![Create Account Online Presence](./create-account-5.png)
+
+6. For the *Contact Preferences* section, select the way you want to receive communication i.e. via **Email**, **Post**, **SMS** or **Telephone**. Each category contains five different options named **Unspecified**, **Admin Only**, **Fundraising Only**, **All** and **None**. Select any category. For details, refer to the Contact Preferences(link to page) page.
+
+:::note Important
+You can select two categories at one time by using the **Opt-In** and **Opt-Out** options.
+:::
+
+![Create Account Contact Preferences](./create-account-6.png)
+
+7. Next, if you want to add a *Gift*, click **Eligibility** as *Yes*. Then add the **Declaration Date**, **Valid from**, choose **Declaration Source** and attach the source files if needed. For details, refer to Gift Aid Section(link).
+
+![Create Account Gift Aid](./create-account-7.png)
+
+8. In the *interactions* section, assign any **Touchpoint** for analytics purposes. Select the **Date** and **Time**. For more information, go to Interactions Section(link).  
```

Comment:
I don't think it adds much value putting this. I would instead just put what the section is for and refer to the link. So something like:

If you wish to track the source of the account for analytics purposes you can do that by assigning an initial touchpoint to the account. If unsure, leave this blank.

Also I can't merge a PR into production with (link) as that doesn't do anything.

Reply:
I just added the (link) word to only make me remember where I need to add internal links to other docs.